lib/opensusebasetest.pm

```diff
@@ -708,6 +708,18 @@ sub post_fail_hook {
         }
     }
 
+    if (get_var('FULL_LVM_ENCRYPT') && get_var('LVM_THIN_LV')) {
```

Comment:
For me it looks like that we could check if `LVM` is set to 1 here, as this will be useful for all of them I guess. We could check with https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/variables.md 
But it's also fine for now.